signoff/__init__.py

```diff
@@ -200,6 +200,10 @@ def format_signoff_short(signoff_pkg, local_pkg, options):
     elif options.show_uninstalled and local_pkg:
         formatted += click.style(" (installed)", bold=True, fg="blue")
 
+    # show known bad indicator
+    if signoff_pkg["known_bad"]:
+        formatted += click.style(" (bad)", bold=True, fg="red")
```

Comment:
Can we append ` (known bad)` here instead? It's not like we're tight on line length here, and it's more clear what it means.

Reply:
Fixed in b74554b